frontend/src/app/sessions/user-sessions-wrapper/create-sessions/create-provisioned-session/create-provisioned-session.component.html

```diff
@@ -0,0 +1,144 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB InfraGO AG and contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+<div class="wrapper flex size-full flex-col">
+  <div class="mb-2 flex min-h-7 gap-2">
+    <h2 class="flex items-center text-xl font-medium">
+      @if ((projectWrapperService.project$ | async)?.type === "training") {
+        Training Provisioning
+      } @else {
+        Session Provisioning
+      }
+    </h2>
+    <span
+      class="flex items-center gap-1 rounded border bg-green-800 px-2 py-1 text-sm text-white"
+      ><app-mat-icon class="contents" size="15px">experiment</app-mat-icon
+      ><span>Experimental</span></span
+    >
+  </div>
```

Comment:
As this feature is currently marked as experimental, it might make sense to put it below the list of models on non-training projects until we've tested it more thoroughly in production. We could also use the new beta-testing feature to conditionally enable it for non-training projects.

Reply:
I'll keep the position to make the new feature visible, but limit visibility to beta-users only.